src/decision/src/LidarDecision.cpp

```diff
@@ -29,6 +29,8 @@ LidarDecision::LidarDecision(int argc, char **argv, std::string node_name) {
     // Get Param(s)
     if (!public_nh.param<float>("max_obstacle_angle_diff", max_obstacle_angle_diff, (float)M_PI/36))
         ROS_WARN("max_obstacle_angle_diff not set, defaulting to %f", max_obstacle_angle_diff);
+    if (!public_nh.param<float>("max_obstacle_angle_diff", max_obstacle_angle_diff, (float)4))
+        ROS_WARN("max_obstacle_distance_diff not set, defaulting to %f", max_obstacle_angle_diff);
```

Comment:
Probably just a copy-paste mistake, but I think you mean to try and get the "max_obstacle_distance_diff" param, rather then "max_obstacle_angle_diff" in this statement.